DataStructures/Dictionaries/OpenAddressingHashTable.cs

```diff
@@ -0,0 +1,225 @@
+﻿/***
+ * Open Addressing Hash Table
+ * 
+ * Implementation of Open Addressing Hash Table using
+ *      - Linear Probing (coming soon)
+ *      - Quadratic Probing (coming soon)
+ *      - Double Hashing
+ *      
+ * Author: Samuel Kenney
+ * Created: 6/7/2017
+ * Last Modified: 6/13/2017
+ * 
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using DataStructures.Common;
+using DataStructures.Hashing;
+using System.Threading.Tasks;
+
+namespace DataStructures.Dictionaries
+{
+    /// <summary>
+    /// Open Addressing Data Structure
+    /// </summary>
+    /// <typeparam name="TKey"></typeparam>
+    /// <typeparam name="TValue">will always be an integer since it maps to an index</typeparam>
+    public class OpenAddressingHashTable<TKey>
```

Comment:
Hi @aalhour, I have had a chance to review your comments and the other hash table implementations. From my understanding and research of an Open-Addressing Hash Table, the key will map to a value which will always be an int. This value corresponds to the index in the Table. As of now, my implementation returns a value that the key is linked to but it is not explicitly saved in the table. I can implement the Dictionary interface but I do not think that I need a generic TValue type. Shouldn't it only be an int since that is what the key is being mapped to? It does not seem that the other hash tables really even use the value. What are your thoughts on this? I can explicitly map to the int that it hashes and then store it in the table.